frontend/app/components/Home.tsx

```diff
@@ -1,9 +1,12 @@
-import React, { useEffect, useState } from 'react';
-import { Alert, Grid, Typography } from '@mui/material';
+import React, { useState } from 'react';
+import { useQuery } from 'react-query';
+import { Alert, Box, Grid, LinearProgress, Typography } from '@mui/material';
 import { styled } from '@mui/material/styles';
 
+import ErrorsDisplay from './ErrorsDisplay';
 import HeaderAppBar from './HeaderAppBar';
 import ToolCard from './ToolCard';
+import * as api from '../api';
```

Comment:
Maybe nitpick but Airbnb style guide says [do not use wildcard imports](https://github.com/airbnb/javascript#modules--no-wildcard). This this really needed here? Is this common for some situations?

Reply:
This is a pattern I followed from ROHQ. It's kind of nice to see in this case which functions are coming from the `api` module (and which by extension depend on external connections), but I don't mind changing it.

Reply:
OK, changed it, see https://github.com/tl-its-umich-edu/canvas-app-explorer/pull/196/commits/124b5de9e40f5a9931e7c9745f7a515b6e18ab4b